### PR TITLE
*: non-unique index doesn't need store value, write a '0' to reduce space

### PR DESCRIPTION
**Pull request by tiancaiamao**
this can save 9 bytes for each unique index

@zimulala @coocood @disksing @shenli

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by coocood** [comment]
non-unique index doesn't need value.

File: table/tables/index.go [hunk 1]
```diff
@@ -165,8 +165,7 @@ func (c *index) Create(rm kv.RetrieverMutator, indexedValues []types.Datum, h in
 		return 0, errors.Trace(err)
 	}
 	if !distinct {
-		// TODO: reconsider value
-		err = rm.Set(key, []byte("timestamp?"))
+		err = rm.Set(key, []byte("0"))
```
Comment (ngaut) [inline table/tables/index.go]: Please add comments.

**Comment by disksing** [comment]
LGTM

**Pull request merged**